rules/REQUEST-930-APPLICATION-ATTACK-LFI.conf

```diff
@@ -31,8 +31,8 @@ SecRule TX:DETECTION_PARANOIA_LEVEL "@lt 1" "id:930012,phase:2,pass,nolog,skipAf
 # util/regexp-assemble/data/930100-dots.data.
 # To update the regular expression:
```

Comment:
I have only one comment: here you left this text (it's okay for me), but at every other rules the new format is this:
```
# To update the regular expression run the following shell script
```
May be it would be fine to replace here too - just for the case, if someone wants to parse the whole text...

Reply:
Yes of course. Thanks for pointing it out.